src/translations/de.translation

```diff
@@ -3922,10 +3922,6 @@ Wenn Sie es vorziehen, können Sie &quot;Schenkungseingabe&quot; deaktivieren un
         <label>Statusprüfungen für Daten in Datensätzen deaktivieren</label>
         <name>healthLabelDisableRecordBatch</name>
     </customLabels>
-    <customLabels>
-        <label>Wenn sie aktiviert ist, prüft die Statusprüfung lediglich Ihre NPSP-Konfigurationseinstellungen auf Probleme.</label>
```

Comment:
@force2b , should we remove translations for the changed custom label, or update the label as following so it is updated by the translation team later on?
```
<label><!-- healthLabelDisableRecordBatchDescription --></label>
```

Reply:
Great question. Looking at the label change, it's not a fundamental change in the meaning of the text. It's really just adding an extra sentence to the end. Given that I think we should leave the current translation in place. It'll get automatically updated in the next translation round.